### ISSUE TITLE
有关unet的问题

### ISSUE THREAD

**Issue by insoliang** [PaddlePaddle/PaddleSeg#1214]
您好，我想请问下为什么相同的数据，使用paddleseg内的基础unet网络和在github上找的其他人搭建的unet比，训练的时间会比较慢呢？
paddleseg的unet，是作者提出的最原始的unet吗？还是对网络做了一些其他的修改与改进呢？

**Comment by wuyefeilin** [comment]
具体慢多少呢， 另外你找的其他搭建的unet是哪一个？比较过两者的差别吗？

**Comment by insoliang** [comment]
感谢回复，我用的2080s，50个epoch，相同配置，paddleseg大概需要一晚上，另找的unet就是最原始的unet网络，大概训练30分钟就结束了，所以想问下paddleseg是基于原始的unet做了改进麻？

**Comment by insoliang** [comment]
我用其他的代码在相同的epoch和batchsize下训练的模型虽然都比paddleseg快很多，但是最终的性能指标都不如paddleseg，想问一下您paddleseg是在训练的过程中加入了一些优化算法吗？如果是这样导致和其他网络的区别的话，我可以实现利用paddleseg训练最原生的网络吗（不加入任何其他优化算法）？目前我的工作是需要做一些对比实验，所以需要对比最原生的unet和fcn等网络，谢谢！

**Comment by insoliang** [comment]
> 具体慢多少呢， 另外你找的其他搭建的unet是哪一个？比较过两者的差别吗？

您好，您还在吗？

**Comment by wuyefeilin** [comment]
你看一下图象输入的尺寸是否一致

**Comment by insoliang** [comment]
> 你看一下图象输入的尺寸是否一致

我统一resize成256*256了，
这是我的config文件

batch_size: 4
iters: 50000

optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

train_dataset:
  type: Dataset
  dataset_root: data/dataset 8.2/all
  train_path: data/dataset 8.2/all/train_list.txt
  num_classes: 3
  transforms:
    - type: Resize
      target_size: [256, 256]
    - type: Normalize
  mode: train

val_dataset:
  type: Dataset
  dataset_root: data/dataset 8.2/all
  val_path: data/dataset 8.2/all/val_list.txt
  num_classes: 3
  transforms:
    - type: Normalize
  mode: val

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.01
  power: 0.9
  end_lr: 1.0e-5

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

model:
  type: FCN
  backbone:
    type: HRNet_W18
    align_corners: False
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/hrnet_w18_ssld.tar.gz
  num_classes: 3
  pretrained: Null
  backbone_indices: [-1]

**Comment by wuyefeilin** [comment]
两者的图象输入都是[256, 256]吗？ 另外你可以增大--num_workers减少数据读取的时间

**Comment by wuyefeilin** [comment]
两者的图象输入都是[256, 256]吗？ 另外你可以增大--num_workers减少数据读取的时间
